### PR TITLE
docs: update milestones.

### PR DESCRIPTION
**Pull request by mpaulucci**
**Motivation**
Update the milestones based on the latest planning

**Description**
- Expanded text to explain better each milestone.
- Updated the RPC endpoints.
- Added a milestone related to block building.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,37 +78,64 @@ kurtosis enclave stop lambdanet ; kurtosis enclave rm lambdanet
...
+- `eth_blockNumber`
+- `eth_call`
+- `eth_chainId`
+- `eth_createAccessList`
+- `eth_estimateGas`
+- `eth_feeHistory`
+- `eth_getBalance` (at head block)
 - `eth_getBlockByHash`
 - `eth_getBlockByNumber`
-- `eth_blockNumber`
```
Comment (fmoletta) [inline README.md:87]: why remove this? It is currently supported

Comment (mpaulucci) [reply README.md:87]: I just sorted them alphabetically

File: README.md [hunk 2]
```diff
@@ -78,37 +78,64 @@ kurtosis enclave stop lambdanet ; kurtosis enclave rm lambdanet
...
 ### Milestone 1: RPC Node
-Add support to participate in a Cancun localnet as a read-only node.
+Add support to follow a post-Merge localnet as a read-only RPC Node. This first milestone will only support a canonical chain (every incoming block has to be the child of the current head). 
 
 RPC endpoints
-- `engine_newPayloadV3` (partially)
+- `engine_newPayloadV3` (excl. block building)
+- `eth_blobBaseFee`
+- `eth_blockNumber`
+- `eth_call`
```
Comment (ElFantasma) [inline README.md:87]: (at head block) 😞

**Review by ElFantasma — APPROVED** [review]
LGTM

**Pull request merged**